freeipa-api/src/main/java/com/sequenceiq/service/api/doc/ModelDescriptions.java

```diff
@@ -11,14 +11,10 @@
     public static final String CHILD_ENVIRONMENT_CRN = "CRN of the child environment";
     public static final String CLUSTER_CRN = "CRN of the cluster";
     public static final String INSTANCE_ID = "ID of the instance";
+    public static final String FORM_FACTOR = "Form factor of the resource";
```

Comment:
this could be a bit more descriptive I think. Maybe explaining what do the values mean or similar